accord-core/src/main/java/accord/topology/TopologyManager.java

```diff
@@ -498,7 +508,7 @@ public Topologies withUnsyncedEpochs(Unseekables<?> select, long minEpoch, long
         return withSufficientEpochs(select, minEpoch, maxEpoch, epochState -> epochState.syncComplete);
     }
 
-    public Topologies withOpenEpochs(Unseekables<?> select, Timestamp min, Timestamp max)
+    public Topologies withOpenEpochs(Unseekables<?> select, EpochSupplier min, EpochSupplier max)
```

Comment:
this was updated for `ListStore` as it works with historic epochs; so rather than define a full `Timestamp`, using `EpochSupplier` allowed `() -> epoch` as well in this case